client/src/assets/player/utils.ts

```diff
@@ -170,9 +170,10 @@ function secondsToTime (seconds: number, full = false, symbol?: string) {
   return time
 }
 
-function buildVideoOrPlaylistEmbed (embedUrl: string) {
+function buildVideoOrPlaylistEmbed (embedUrl: string, embedTitle: string) {
```

Comment:
I think you need to escape the `embedTitle`. You can use the `escapeHTML` from `server/helpers/core-utils` after moving the function in `shared/core-utils/renderer.html.ts`

Reply:
hi @Chocobozzz can you check if i made correct changes ?